core/modules/filter/filter.theme.inc

```diff
@@ -198,3 +198,21 @@ function theme_filter_tips($variables) {
 
   return $output;
 }
+
+function theme_filter_format_editor_image_form($variables) {
+  $form = $variables['form'];
+  
+  $output = '<div class="image-form-wrapper"><div class="image-form-left">';
+  $output .= backdrop_render($form['library_view']);
+  $output .= '</div><div class="image-form-right">';
```

Comment:
We shouldn't use classes like "left" or "right" because on an RTL language, these will be reversed. Instead name these based on their contents, e.g. "editor-image-library" and "editor-image-fields".

Reply:
Understood. I have changed the class names as you suggest.